src/bin/pg_waldump/pg_waldump.c

```diff
@@ -939,6 +958,7 @@ main(int argc, char **argv)
 		{"stats", optional_argument, NULL, 'z'},
 		{"save-fullpage", required_argument, NULL, 1},
 		{"save-records", required_argument, NULL, 2},
+		{"load-records", required_argument, NULL, 3},
```

Comment:
nit: load-ps-records. This is not loading records with --save-records. 

probably we can implement get-page so that load-records can load records in both places.